mantis-runtime/src/test/java/io/mantisrx/runtime/executor/TestJobParameterized.java

```diff
@@ -86,6 +86,10 @@ public Integer call(Integer t1) {
                 }, new ScalarToScalar.Config<Integer, Integer>()
                         .codec(Codecs.integer()))
                 .sink(new Sink<Integer>() {
+                    @Override
+                    public void init(Context context) {
+                        System.out.println("sink init called");
```

Comment:
Did you meant to remove this?

Reply:
No that is a test class, added it to show it gets called before the call method.